### PR TITLE
creative change important

### PR DESCRIPTION
**Pull request by sareh88**

### CODE REVIEW

**Comment by josearmandojacq** [comment]
hello

**Comment by aemal** [comment]
leave a comment here... for testing...

**Comment by hassoun86** [comment]
wowwww

File: lib/models/user.model.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ const contact = new Schema({
 	avatarURL: { type:String, required:true },
 	userId: { type:Schema.Types.ObjectId, required:true, ref: 'User'}
 })
-//2
+//3 creative
 const userSchema = new Schema({
 	emailAddress: { type: String , lowercase: true, unique : true},
   password: String, // Do not forget, this is an SHA-512 Digest, NOT pure password
```
Comment (aemal) [inline lib/models/user.model.js]: this needs a tab

Comment (numbing) [reply lib/models/user.model.js]: :) just for test

File: lib/models/user.model.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ const contact = new Schema({
 	avatarURL: { type:String, required:true },
 	userId: { type:Schema.Types.ObjectId, required:true, ref: 'User'}
 })
-//2
+//3 creative
```
Comment (aemal) [inline lib/models/user.model.js]: make c of Creative capital

**Review by aemal — APPROVED** [review]

**Pull request merged**